src/GuzzleHttp/Cache/DoctrineAdapter.php

```diff
@@ -43,7 +43,7 @@ public function save(RequestInterface $request, ResponseInterface $response)
         $data = [
             'status' => $response->getStatusCode(),
             'headers' => $response->getHeaders(),
-            'body' => (string)$response->getBody(),
+            'body' => (string) $response->getBody(),
```

Comment:
This change should be reverted. We don't want to pollute this PR with CS-related fixes.